deps.bzl

```diff
@@ -6935,3 +6936,15 @@ def install_static_dependencies(workspace_name = "buildbuddy"):
         sha256 = "85b1c2591274422234955e906aee39e6f793a88a74f3efc49a1852a0646ce08f",
         urls = ["https://storage.googleapis.com/buildbuddy-tools/binaries/clang-format/clang-format-14_linux-x86_64"],
     )
+
+    http_file(
+        name = "com_github_buildbuddy_io_podman_static_podman-linux-amd64",
+        urls = ["https://github.com/buildbuddy-io/podman-static/releases/download/buildbuddy-{podman_version}/podman-linux-amd64.tar.gz".format(podman_version = PODMAN_VERSION)],
+        sha256 = "ac14152d2ef5cb25abbd615893cd56355f951f075a7ae663f0b2c1bc5d3fb77e",
```

Comment:
Won't the hash here and below change with the version? And if so, should it be defined in the same place as `PODMAN_VERSION` to reduce confusion?